docs/en/revocation-lists.rst

```diff
@@ -57,7 +57,7 @@ Functional Requirements
 - revoke a Digital Credential when the following circumstances occur:
 
   - the Digital Credential requires to be updated, whenever one or more attributes are changed; in this case the User will request a new issuance for that Digital Credential;
-  - the Wallet Instance that holds the Digital Credential was issued is revoked;
+  - the holder request such revocation due to loss or replacement of cryptographic key material which the issued Digital Credential is bound to;
```

Comment:
```suggestion
  - the Holder request such revocation due to loss or replacement of cryptographic key material which the issued Digital Credential is bound to;
```

how it can request the revocation if it cannot prove to be the legitimate possessor of that credential?

Reply:
The only way could be making the request directly to the Issuer which must authenticate the User.

Reply:
shall we say this in the text?